tests/phpunit/tests/admin/wpSiteHealth.php

```diff
@@ -499,4 +499,80 @@ public function data_object_cache_thresholds() {
 			array( 'alloptions_bytes', 1000 ),
 		);
 	}
+
+	/**
+	 * Tests get_test_autoloaded_options() when autoloaded options less than warning size.
+	 *
+	 * @ticket 61276
+	 *
+	 * @covers ::get_test_autoloaded_options()
+	 */
+	public function test_wp_autoloaded_options_test_no_warning(): void {
```

Comment:
@joemcgill @mukeshpanchal27 Query: I see there are several return type declarations in this PR. Is that something now being done in other parts of Core? If so, it is just a new convention or is there a Make post that outlines any new conventions that are usable?

Reply:
I will remove it as it's an improvement based on PHPStan suggestions.

Reply:
Removed in https://github.com/WordPress/wordpress-develop/pull/6622/commits/99e09a6ae5572e59dcbef2ee2dca83857da6fd9a

Reply:
Thanks @mukeshpanchal27 and @joemcgill! I genuinely wasn't sure if there'd been changes so thanks to both for confirming!